src/blackbox_decode.c

```diff
@@ -1073,6 +1075,50 @@ void resetParseState() {
     seriesStats_init(&looptimeStats);
 }
 
+void writeLogHeaderLine(const char *lineStart, const char *lineEnd) {
+    if (lineEnd - lineStart < 3) {
+        return;
+    }
+
+    if (*lineStart != 'H') {
+        return;
+    }
+
+    if (*(lineStart + 1) != ' ') {
+        return;
+    }
+    const char *separatorPos = strchr(lineStart, ':');
+    if (separatorPos == NULL || separatorPos > lineEnd) {
+      return;
```

Comment:
Please use indentation of 4 consistently (only showing this once)
```suggestion
        return;
```